maps/src/MapMockObserver.cxx

```diff
@@ -17,7 +17,8 @@ class MapMockObserver : public G3Module {
 public:
 	MapMockObserver(std::string pointing, std::string timestreams,
 	    double band, G3SkyMapConstPtr T, G3SkyMapConstPtr Q,
-	    G3SkyMapConstPtr U, std::string bolo_properties_name);
+	    G3SkyMapConstPtr U, std::string bolo_properties_name,
+	    std::string bolos, bool interp);
```

Comment:
Shouldn't "bolos" be a list/vector of strings rather than one string?

Reply:
It's a key to a `G3VectorString` object that's stored in each Scan frame, presumably because the list can change frame-by-frame?

Reply:
Ah -- that isn't in the docs. Why would it change frame-by-frame? It doesn't in the DAQ, which is what this is trying to simulate.

Reply:
The simstubs have a key in them that's constructed from some timestream map that is put together after a few timestream analysis steps (mainly glitch finding, it seems).  I wouldn't expect the bolometer list to change between frames, but I also don't know why the simstubs would be written this way if that field is completely duplicated...

Reply:
It's an archaism, I think, from the very original version of the map code. The more I think about it, the more I think I would like to leave it out -- unless the number of glitchy detectors is enormous, in which case the observation is presumably cut, simulating them doesn't seem like a huge waste of CPU and it matches the simulation outputs more closely to the real instrument, while making the stubs simpler at the same time.

Reply:
Agreed.  I'll remove this feature and update the mapmaker script appropriately.